packages/react-router/lib/components.tsx

```diff
@@ -109,16 +109,6 @@ export function RouterProvider({
   );
 }
 
-export interface DataMemoryRouterProps {
```

Comment:
Noticed this was leftover from the `DataMemoryRouter -> RouterProvider` refactor